irohad/multi_sig_transactions/transport/mst_transport_grpc.hpp

```diff
@@ -56,10 +54,20 @@ namespace iroha {
                      ConstRefState providing_state) override;
 
      private:
+      /**
+       * Flat map transport transactions to shared model
```

Comment:
Is the term "flat map" accurate here? As far as I know, flat mapping is about mapping to nested collections and then unwrapping elements to a collection "higher". But here we map a collection of txs to a collection of txs, and I think there's no need to get into attention intermediate `Result`-s in the header file comment

Reply:
Flat map takes a list of elements and a function, which returns a collection of elements and flattens it. This is what this function does - it takes each transaction, performes validation and mapping to shared_model, and either returns an empty value if validation fails (collection of zero elements), or a transaction (collection of one element). These collections are then flattened to a list of transactions.